### PR TITLE
Narrow fix for tintColor and repeat images

### LINKED ISSUE #7205: Images rendered with ReactImageBrush disappear in multiple monitors
## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. Run RNTester with multiple monitors on host device.
2. Open the Image component example
3. Scroll to images with tintColor example
4. Move app to a different monitor 

## Expected Results
Tinted images should still be rendered on the screen, but instead they disappear. I suspect this has something to do with not registering for events related to the CanvasDevice / CompositionGraphicsDevice to trigger redrawing when needed. Not sure if it's expected that the XamlCompositionBrushBase is expected to handle this for you (and thus a bug with that component) or if we need to create something similar to the ImageLoader here: https://github.com/microsoft/WindowsCompositionSamples/blob/master/SamplesCommon/SamplesCommon/ImageLoader/ImageLoader.cs 

## Snack, code example, screenshot, or link to a repository:

https://user-images.githubusercontent.com/1106239/108873791-7b279500-75c9-11eb-8c50-f7c038a7e68d.mp4

### PR DESCRIPTION
**Pull request by rozele**
There is a [bug](https://github.com/microsoft/Win2D/issues/809) in GaussianBlurEffect from Win2D that causes images to disappear when moving the app between screens. Since the ReactImageBrush always created the GaussianBlurEffect (even when `blurRadius` was 0) whenever `resizeMode=repeat` or `tintColor` was used on the Image, this meant that Images that only used `resizeMode` or `tintColor` suffered from the same bug.

This change ensures that GaussianBlurEffect is only used when the `blurRadius` prop is non-zero.

Fixes #7205

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/7222)

### CODE REVIEW

File: vnext/Microsoft.ReactNative/Views/Image/ReactImageBrush.cpp [hunk 1]
```diff
@@ -100,7 +101,9 @@ void ReactImageBrush::UpdateCompositionBrush(bool forceEffectBrush) {
       // "You can compute the blur radius of the kernel by multiplying the standard deviation by 3.
       // The units of both the standard deviation and blur radius are DIPs.
       // A value of zero DIPs disables this effect entirely."
-      compositionBrush.Properties().InsertScalar(BlurBlurAmount, m_blurRadius / 3);
+      if (m_blurRadius > 0) {
```
Comment (acoates-ms) [inline vnext/Microsoft.ReactNative/Views/Image/ReactImageBrush.cpp:104]: I dont know this code.  But can blurRadius be non-zero, then on a subsequent render be zero?  -- Would this value need to be removed in that case?

Comment (acoates-ms) [reply vnext/Microsoft.ReactNative/Views/Image/ReactImageBrush.cpp:104]: Actually looks like this function is creating the compBrush.. so probably not an issue here.

Comment (rozele) [reply vnext/Microsoft.ReactNative/Views/Image/ReactImageBrush.cpp:104]: If it changed from non-zero to zero then we'll force the creation of a new effect brush, so the properties will be reset.

Comment (rozele) [reply vnext/Microsoft.ReactNative/Views/Image/ReactImageBrush.cpp:104]: https://github.com/microsoft/react-native-windows/pull/7222/files#diff-85a95b36797ca57b21e2e39c5e678c0ab8a0026d716b1dce56de2311e11f3143R49

**Comment by asklar** [comment]
@rozele  FYI we don't use Win2D, we have our own copy of the effect: `Microsoft.ReactNative\Views\cppwinrt\Effects.idl`
We should probably be fixing the effect instead (regardless of whether image was unnecessarily creating the effect for tint/repeat, which I'm not sure if it was intentional).

**Comment by rozele** [comment]
@asklar agree the effect definitely needs to be fixed, but as you said this fix has it's own merits of removing unnecessary effects.

I looked at what's happening with the GaussianBlurEffect, and all it seems to be doing is forwarding props to D2D, so my thought was that the bug was with D2D, and thus not fixable here. Thought maybe Win2D might have some better idea of how to work around or fix.

**Comment by marlenecota** [comment]
Thanks @rozele ! At the time, I was having issues daisy chaining the effects when the blur value flipped between zero and non-zero, so my solution was to always have blur on as the base if any of the other effects were at play since a value of zero would turn it off. This is a much cleaner solution and it looks like you already validated this scenario.

**Review by marlenecota — APPROVED** [review]

**Pull request merged**